artview/core/common.py

```diff
@@ -229,25 +229,37 @@ def handleClick(button):
 # Option methods #
 ##################
 
+
+def float_or_none(text):
+    if text=="None":
+        return None
+    else:
+        return float(text)
+
```

Comment:
A possibly more robust test for the above could be something like
```
try:
    return float(text)
except:
    return None
```

This might handles cases of misspelling None or some other weird value or type